docs/snippets/process-out-env.nf

```diff
@@ -1,6 +1,6 @@
 process myTask {
     output:
-    env FOO
+    env 'FOO'
```

Comment:
This is expected to work without quotes

Reply:
The formal syntax requires quotes, otherwise the parser will treat it as a variable when it's really a name. Also the quotes are more readable

I will add a note about this restriction to the other PR

Reply:
Actually, this was designed to work primarily without the need for quotes 

https://github.com/nextflow-io/nextflow/blob/500f49c0911bb33e3fa0f765df4a20da07a8f761/modules/nextflow/src/test/groovy/nextflow/script/params/ParamsInTest.groovy#L335-L337

Reply:
both are supported, but the one without quotes is problematic because it looks like a variable declaration when in reality it is not

Reply:
at nextflow level you mean?

Reply:
yes

Reply:
Make sense. Agree